SRC/classes/HelloWorldTestClass.cls

```diff
@@ -0,0 +1,17 @@
+@isTest 
+private class HelloWorldTestClass {
+    static testMethod void validateHelloWorld() {
```

Comment:
and method also should mathc the method u created in class
Ex: method Name : applyDiscount
     test Method Name : applyDiscountTests